initial work

donovaly opened the pull request:
initial work on the docs, just check it out and please tell me if you could pull this change

donovaly committed:
initial work

donovaly committed:
- weitere Arbeit
- HTML Ausgabe hinzugefügt

donovaly committed:
again more work

donovaly committed:
Handbuch: convert to format of LyX 2.1.x

it was a mistake to use an unreleased version of LyX for editing.

donovaly committed:
split Handbuch into its chapters

- to be able to get a separate HTML file for each chapter
- was requested by Emanuel

donovaly committed:
move .rst file into attic

donovaly committed:
- describe the file handling
- rename an image file

donovaly committed:
- describe Participants handling (import will follow)
- add appendix to the main Handbuch document
- rename a symbol

donovaly committed:
add some screenshots

donovaly committed:
describe messages and countdowns

- correct 2 broken cross-references
- clarify group settings

donovaly committed:
- describe import of participants via CSV files

donovaly committed:
update HTML output according to previous commit

donovaly committed:
- add info how to create CSV files easily

donovaly committed:
- describe agenda in Tutorial

donovaly committed:
- add missing LyX file for the Tutorial

donovaly committed:
update HTML output

donovaly committed:
describe agenda in the detail

donovaly committed:
add info about agenda attachments and start description of elections

donovaly committed:
describe elections in the tutorial completely

donovaly committed:
start description of elections in detail

donovaly committed:
finish description of elections

donovaly committed:
begin description of motions

also rename the administration tutorial to separate it from the new tutorial for normal participants

donovaly committed:
renaming part 2

donovaly committed:
update HTML output

donovaly committed:
continue description of motions

donovaly committed:
some more descriptions of the currently very buggy motion system

donovaly committed:
continue description of motions

donovaly committed:
- update to LyX 2.2 (http://wiki.lyx.org/LyX/NewInLyX22)

- continue the description of motions
- add a note about user groups

donovaly committed:
- add some hints as discussed in bug reports

donovaly committed:
continue description of motions

donovaly committed:
add info about an option

donovaly committed:
Add a missing cross reference.

donovaly committed:
correct an image scaling factor

donovaly committed:
describe projector settings

emanuelschuetze commented:
@donovaly Good work! (I have not read the whole documentation but first view looks good)
Please note that you should rebase all your commits together to one commit. Use `git rebase -i HEAD~<number of last commits>` (test it e.g. with `git rebase -i HEAD~2`; then write for the second commit a `s` to squash it, see also here http://de.gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).

emanuelschuetze commented:
Suggestion: Make a "backup branch" before you play with git rebase...

donovaly committed:
describe the OpenSlides tools how to document a meeting

donovaly committed:
add image files

donovaly committed:
- change fileformat of the LyX files

Back to the one of the old (and also no longer supported LyX 2.1.x series). If possible please use LyX 2.2.x to edit the docs.

Also add some info how to build the docs.

emanuelschuetze commented:
I moved this initial work into a new pull request (see #4). Closed here.
